dereliction.json

```diff
@@ -0,0 +1,40 @@
+{
+	"info": { "version": "3.0.0-beta3", "last_update": 1628353970 },
+	"name": "Dereliction Requests",
+	"collectionId": "ccmap/ump/exclusion_zones",
+	"presentations": [
+		{
+			"name": "Dereliction Requests",
+			"style_base": {
+				"color": "$color|#ffff00",
+				"fill_pattern": "yellow_stripes",
+				"fill_opacity": 0.7,
+				"opacity": 0.7,
+				"label": null,
+				"stroke_width": 0
+			},
+			"style_highlight": {
+				"opacity": 1,
+				"fill_opacity": 1,
+				"label": "[Dereliction]\nBy: $madeby\nExpires: $dateofexpiration\nReason: $reason",
```

Comment:
I think this doesn't work, IIRC only `"$blah"` and `"$blah|abc"` are supported. Did you try this out?